package.json

```diff
@@ -13,7 +13,7 @@
   "io_bpd_winning_transactions": "https://raw.githubusercontent.com/pagopa/io-services-metadata/master/bonus/specs/bpd/winning_transactions.json",
   "io_bpd_winning_transactions_v2": "https://raw.githubusercontent.com/pagopa/io-services-metadata/master/bonus/specs/bpd/winning_transactions_v2.json",
   "io_eu_covid_cert": "https://raw.githubusercontent.com/pagopa/io-backend/v7.16.0/api_eucovidcert.yaml",
-  "pagopa_api": "https://raw.githubusercontent.com/pagopa/io-services-metadata/master/pagopa/pm/spec.json",
+  "pagopa_api": "https://raw.githubusercontent.com/pagopa/io-services-metadata/IA-166-update-pagopa-spec/pagopa/pm/spec.json",
```

Comment:
I guess this PR depends on https://github.com/pagopa/io-services-metadata/pull/424 ?

Reply:
Nope
It depends on https://github.com/pagopa/io-services-metadata/pull/422 that is already merged
I guess you are talking about this other PR https://github.com/pagopa/io-app/pull/3265